app/helpers/fhir_helper.rb

```diff
@@ -312,48 +350,66 @@ def from_bool_extension(patient, extension_id)
 
   def to_bool_extension(value, extension_id)
     value.nil? ? nil : FHIR::Extension.new(
-      url: "http://saraalert.org/StructureDefinition/#{extension_id}",
+      url: "#{SARA_BASE_URL}/StructureDefinition/#{extension_id}",
```

Comment:
No that's a good point, I guess I can't really think of a situation where these would change. In fact we'd want to avoid changing this URL almost definitely. So I will revert this.

Reply:
Gone in https://github.com/SaraAlert/SaraAlert/pull/739/commits/f6c9163b07025e6160242681f5dd6e85e3b67ef1.